pkg/window/strategy/sliding/sliding.go

```diff
@@ -161,6 +161,8 @@ func (s *Sliding) RemoveWindows(wm time.Time) []window.AlignedKeyedWindower {
 		if win.EndTime().Before(wm) {
 			s.entries.Remove(e)
 			closedWindows = append(closedWindows, win)
+		} else {
```

Comment:
This for loop will only run for 1 iteration, is that expected?

Reply:
this for loop will run until you find a window with end time > watermark